crates/workspace/src/workspace.rs

```diff
@@ -630,7 +631,9 @@ impl Workspace {
                     this.collaborator_left(*peer_id, cx);
                 }
 
-                project::Event::WorktreeRemoved(_) | project::Event::WorktreeAdded => {
+                project::Event::WorktreeRemoved(_)
+                | project::Event::WorktreeAdded
+                | project::Event::WorktreesSorted => {
```

Comment:
Current behavior in `main` seems to restore the sorted order without this change, so not sure why do we need to react on this after sorting the worktree list?

Reply:
This is needed to trigger the project panel to update when running the action in the command palette with an unsorted project list open.

Reply:
Sorry, I still find this confusing: on this level, `this` is `workspace` and the code says that when the workspace items had been sorted, it needs to update the window title and serialize the workspace into the DB.

While there could be some side effects that force the project panel to update, it is supposedly already happening due to https://github.com/zed-industries/zed/pull/11504/files#diff-15641a41f7176cd74f143b22940d1ea6b279b1cb9e6330d8e8dab238118ee992R212 change in the project panel (where it catches the same event and decides to mutate itself somehow)?
I would assume that event handler in the project panel is enough?

Reply:
I'll test it without this and update here 👍🏻